README.md

```diff
@@ -90,6 +91,21 @@ See this project's **[Javadoc](https://spotify-web-api-java.github.io/spotify-we
 
 *A huge thanks to [c-schuhmann](https://github.com/c-schuhmann) for his amazing work on the documentation!*
 
+## Create a Spotify Access Token
+Before using the Java Wrapper for the Spotify Web API, it is required to acquire an access token from Spotify. Follow the steps below to set up your first Spotify app and obtain an access token:
+
+1. **Create a Spotify Developer Account:** If you haven't already, sign up for a Spotify Developer Account at the **[Spotify Developer Dashboard](https://developer.spotify.com/)**.
+2. **Create a new App:** Once you have logged in to the Developer Dashboard, create a new app. You will need to provide some basic information about your app, such as its name and description.
+3. **Set Redirect URI:** A Redirect URI is necessary to complete the form required to create your first app. The Redirect URI is the URI that spotify will redirect users to after they have been authorized. Generally, this may be the homepage of your web application but for testing purposes you can use `http://localhost:8080/callback`.
+4. **Get Client ID and Client Secret:** After creating the app, you'll receive a Client ID and Client Secret. Click on the _settings_ button - the Client ID can be found here. The Client Secret can be found behind the _View client secret_ link.
+5. **Use CURL to get Access Token:** Use the 
+```
+curl -X POST "https://accounts.spotify.com/api/token" \
+     -H "Content-Type: application/x-www-form-urlencoded" \
+     -d "grant_type=client_credentials&client_id=your-client-id&client_secret=your-client-secret"
+```
+6. Congratulations!! You now have an access token that will expire in 60 minutes. Refer to **[Authorization](#Authorization)** to acquire a better understanding of the authorization process.
+
```

Comment:
Here we should just link to https://developer.spotify.com/documentation/web-api/concepts/apps instead of duplicating that page's contents